### PR TITLE
Continue `PaginatorTrait`

### PR DESCRIPTION
**Pull request by billy1624**
## Info
- Continue #306

### CODE REVIEW

File: src/entity/mod.rs [hunk 1]
```diff
@@ -117,3 +117,6 @@ pub use model::*;
 // pub use prelude::*;
 pub use primary_key::*;
 pub use relation::*;
+
+// Imports paginator utility by default making it backward compatible.
+pub use crate::PaginatorTrait;
```
Comment (tyt2y3) [inline src/entity/mod.rs:122]: I think it should belong to `query::*` instead of `entity::*`

Comment (billy1624) [reply src/entity/mod.rs]: Changed on f975223

File: src/query/mod.rs [hunk 2]
```diff
@@ -23,3 +23,6 @@ pub use update::*;
 pub use util::*;
 
 pub use crate::{ConnectionTrait, InsertResult, Statement, UpdateResult, Value, Values};
+
+// Imports paginator utility by default making it backward compatible.
+pub use crate::PaginatorTrait;
```
Comment (tyt2y3) [inline src/query/mod.rs:28]: Not nitpicking, but it actually can fit in the above statement.

Comment (billy1624) [reply src/query/mod.rs:28]: Can ah, just wanna put an explicit comment on it. I can remove the comment if it's unnecessary

Comment (tyt2y3) [reply src/query/mod.rs:28]: It's unnecessary.

**Review by tyt2y3 — APPROVED** [review]

**Pull request merged**